README.rst

```diff
@@ -246,7 +246,7 @@ License
 
 .. |Windows| image:: https://img.shields.io/badge/platform-Windows-brightgreen.svg
 .. |Linux| image:: https://img.shields.io/badge/platform-Linux-brightgreen.svg
-.. |Stable| image:: https://img.shields.io/badge/latest%20stable-0.9.0-green.svg
+.. |Stable| image:: https://img.shields.io/badge/latest%20stable-0.10.0-green.svg
```

Comment:
This seems to be good !! You have to follow commit guide lines!
https://api.coala.io/en/latest/Developers/Writing_Good_Commits.html
Also you have 3 different commits for this.While it's a typo change can be done in one single commit.Try soft reset and then stage and commit followed by force push :smiley: